TarSCM/changes.py

```diff
@@ -219,25 +231,39 @@ def get_changesauthor(self, args):
             return args.changesauthor
 
         # return changesauthor if set by osc
-        if os.getenv('VC_MAILADDR'):
-            logging.debug("Found changesauthor in VC_MAILADDR='%s'",
-                          os.environ['VC_MAILADDR'])
-            return os.environ['VC_MAILADDR']
+        if os.getenv('VC_REALNAME') and os.getenv('VC_MAILADDR'):
+            logging.debug("Found changesauthor in VC_REALNAME='%s' and "
+                          "VC_MAILADDR='%s'" % (os.environ['VC_REALNAME'],
+                                                os.environ['VC_MAILADDR']))
+
+            return (os.environ['VC_REALNAME'] + ' <'
+                    + os.environ['VC_MAILADDR'] + '>')
```

Comment:
Can anyone give some advice here? AFAICT, Python's implied line continuation is being used here because of the parentheses.

``` python
            return (os.environ['VC_REALNAME'] + ' <' +
                    os.environ['VC_MAILADDR'] + '>')
```
---
``` python
            return (os.environ['VC_REALNAME'] + ' <'
                    + os.environ['VC_MAILADDR'] + '>')
```